src/SceneGraphDebugCommandController.ts

```diff
@@ -0,0 +1,291 @@
+// eslint-disable-next-line
+const Telnet = require('telnet-client');
+
+export class SceneGraphDebugCommandController {
+    constructor(public host: string) {
+    }
+
+    private connection: typeof Telnet;
+    private connectionSuccessful = false;
+
+    private shellPrompt = '>';
+    private echoLines = 0;
+    private timeout = 5000;
+    private port = 8080;
```

Comment:
Let's make this configurable (perhaps in a future PR). We already do this for the other debugger ports (i think).

Reply:
I think we only do it for telnet and that was so there was logs from a web-emulator still going into the extension. I think this is very unlikely to need to be changed as this is very os tied. Unlike telnet where it's also the logs dumping grounds.

Reply:
Well it also allows for port forwarding for remote Roku development. But we can deal with that later I guess.

Reply:
We also support the ports for roku-deploy (`packagePort` and `remotePort`).